src/components/landing/GroupLogin.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LandingCard from './LandingCard.jsx';
+import TextInput from '../TextInput.jsx';
+import Button from '../Button.jsx';
+import '../../scss/landing.scss';
+
+/// callBack should be passed from landing page. Used to re-render the landing page
+/// with the default landing page card, which should be the login card.
+
+function GroupLogin({ callBack }) {
+  let input;
+
+  function submitCode() {
+    // TODO: Make login request
+  }
+
+  return (
+    <form onSubmit={submitCode}>
+      <LandingCard title="No account? No problem">
```

Comment:
The `LandingCard` should be the top level element and then have a `<form>` within it.